misc/keygen/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "keygen"
+version = "0.1.0"
+edition = "2021"
+authors = ["demfabris <demfabris@gmail.com>"]
+license = "MIT"
+repository = "https://github.com/libp2p/rust-libp2p"
+keywords = ["peer-to-peer", "libp2p", "networking"]
+categories = ["network-programming", "asynchronous"]
+
+[dependencies]
+structopt = "0.3.26"
+zeroize = "1.5.2"
+serde = { version = "1.0.136", features = ["derive"] }
+serde_json = "1.0.79"
+libp2p = { version = "0.42.2", default-features = false }
```

```suggestion
 libp2p-core = { path = "../../core", default-features = false, version = "0.32.0"}
```

I would suggest only depending on `libp2p-core` and to depend on the version in the relative path.